projects/storefrontapp-e2e-cypress/cypress/integration/regression/product-configurator-vc-interactive.e2e-spec.ts

```diff
@@ -285,3 +286,60 @@ context('Product Configuration', () => {
     });
   });
 });
+
+context('Retract mode for Product Configuration', () => {
+  let configUISettings: ConfiguratorUISettingsConfig;
+
+  beforeEach(() => {
+    configUISettings = {
+      productConfigurator: {
+        addRetractOption: true, // enable retract triggered
+      },
+    };
+    cy.cxConfig(configUISettings);
+  });
+
+  afterEach(() => {
+    configUISettings.productConfigurator.addRetractOption = false; // disable retract triggered
+  });
+
+  describe('Enable retract mode', () => {
```

Comment:
Please re-phrase (because enabling retract mode does not do the retract).
What about 2 tests: 
'Enable retract mode' -> 'should lead to additional retract value displayed'
'Selecting retract mode' -> 'should de-select the currently selected value'

Reply:
Done